rootfs/etc/nginx/lua/test/balancer/sticky_test.lua

```diff
@@ -109,18 +109,18 @@ describe("Sticky", function()
         assert.equal(peer, test_backend_endpoint)
       end)
 
-      it("sets a cookie on the client", function() 
+      it("sets a cookie on the client", function()
         local s = {}
         cookie.new = function(self)
           local test_backend_hash_fn = test_backend.sessionAffinityConfig.cookieSessionAffinity.hash
           local cookie_instance = {
             set = function(self, payload)
               assert.equal(payload.key, test_backend.sessionAffinityConfig.cookieSessionAffinity.name)
               assert.equal(payload.value, util[test_backend_hash_fn .. "_digest"](test_backend_endpoint))
-              assert.equal(payload.path, "/")
+              assert.equal(payload.path, nil)
```

Comment:
can you mock `ngx.var.location_path` at https://github.com/kubernetes/ingress-nginx/blob/f2511909cb113fef364280a3212356e8611e54af/rootfs/etc/nginx/lua/test/balancer/sticky_test.lua#L43 and use the mock value here instead of `nil`?